scripts/ingests/bones_archive/ingest_bones_sources.py

```diff
@@ -32,24 +38,14 @@
 no_sources = 0
 inside_if = 0
 
-# Logger setup
-# This will stream all logger messages to the standard output and
-# apply formatting for that
-logger.propagate = False  # prevents duplicated logging messages
-LOGFORMAT = logging.Formatter(
-    "%(asctime)s %(levelname)s: %(message)s", datefmt="%m/%d/%Y %I:%M:%S%p"
-)
-ch = logging.StreamHandler(stream=sys.stdout)
-ch.setFormatter(LOGFORMAT)
-# To prevent duplicate handlers, only add if they haven't been set previously
-if len(logger.handlers) == 0:
-    logger.addHandler(ch)
-logger.setLevel(logging.INFO)
```

Comment:
I moved the logger setup to `astrodb_utils`.  Notice that `logger` is now imported on line 11